src/Microsoft.Azure.WebJobs.Host/Scale/ITargetScaleMonitor.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    public interface ITargetScaleMonitor : IScaleMonitor
+    {
+        Task<int> GetScaleVoteAsync(ScaleStatusContext context);
```

Comment:
Rather than add a new interface, we should be able to just extend the existing ScaleStatus response returned from the existing IScaleMonitor.GetScaleStatus method. E.g. we could add a new TargetWorkerCount property (defaults to -1). Then for an extension supporting TBS, it can return this along with the ScaleVote it already returns.

Reply:
Adding a public member to a public class is a "potential" breaking change in some cases. I tested the new member with FunctionHost referencing new WebJobs SDK + old extension referencing old WebJobs SDK and it works as expected. 
Do you think it's safe to add a new member to the public class here?

Reply:
Yes it's fine to add a new property to a public type - these types were designed with this in mind to make them extensible, exactly for reasons such as this.